de.fu_berlin.inf.dpp.core/src/de/fu_berlin/inf/dpp/negotiation/InstantOutgoingReferencePointNegotiation.java

```diff
@@ -68,8 +68,8 @@ public void editorActivated(User user, SPath file) {
     private List<StartHandle> stoppedUsers = null;
     private User remoteUser = null;
 
-    public InstantOutgoingProjectNegotiation(final JID peer, //
-        final List<IProject> projects, //
+    public InstantOutgoingReferencePointNegotiation(final JID peer, //
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Constructor has 10 parameters, which is greater than 7 authorized. ([squid:S00107](https://sonarcloud.io/organizations/saros/rules#rule_key=squid%3AS00107))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=saros_complete&pullRequest=259&issues=AWZfKMenDZywvPe4LUfU&open=AWZfKMenDZywvPe4LUfU)

[ISSUE_KEY]: AWZfKMenDZywvPe4LUfU